graph/adjacency/impl/TypeAdjacencyImpl.java

```diff
@@ -27,7 +27,7 @@
 import com.vaticle.typedb.core.graph.edge.Edge;
 import com.vaticle.typedb.core.graph.edge.TypeEdge;
 import com.vaticle.typedb.core.graph.edge.impl.TypeEdgeImpl;
-import com.vaticle.typedb.core.graph.iid.EdgeIID;
+import com.vaticle.typedb.core.graph.iid.EdgeViewIID;
```

Comment:
we're renamed `EdgeIID` to `EdgeViewIID` because it was extremely confusing (even when working with the code extensively) that the EdgeIID is _one_ representation of a `ThingEdge` - name one persisted direction of the edge.